### DISCUSSION TITLE
授权操作不显示已有的权限

### DISCUSSION THREAD

**Discussion by FLVE** [hhyo/Archery#1849] [General]
账号管理页面中显示每个用户所拥有的权限。
![image](https://user-images.githubusercontent.com/46079296/191433451-76f4d90a-ff41-47fd-ad17-c6b796e188b1.png)

点击授权按钮，弹出的页面中已有的权限没有打勾。
![image](https://user-images.githubusercontent.com/46079296/191433515-8d0b7e25-2c32-4b7c-a35c-46bdd1d1f5d5.png)
 
进行授权时，需要明确记得有哪些权限。不能根据已经有哪些权限再进行给与额外的权限。

**Comment by hhyo** [comment]
此处本身就不做自动填充

**Reply by FLVE** [reply]
这个后续有计划做自动填充吗？在授权的时候看到有哪些已经授权了，只用授权额外的权限就好。
